### ISSUE TITLE
PHP warnings with PHP 8.3

### ISSUE THREAD

**Issue by Pimmal** [lansuite/lansuite#1093]
When pressing the menu link start the following errors showing up.
```
PHP Warning: Undefined array key "charset" in netsession.net/httpdocs/inc/Classes/DB.php on line 113
PHP Warning: Undefined array key "mod" in netsession.net/httpdocs/inc/Classes/Security.php on line 29
PHP Warning: Undefined array key "action" in netsession.net/httpdocs/inc/Classes/Security.php on line 30
PHP Warning: Undefined array key "step" in netsession.net/httpdocs/inc/Classes/Security.php on line 31
```
When i´m logged in and go to my profile page https://www.netsession.net/index.php?mod=usrmgr&action=details&userid=1 then the following errors showing up.
```
PHP Warning: Trying to access array offset on null in netsession.net/httpdocs/modules/hardware/plugins/usrmgr_details_tab.php on line 10
PHP Warning: Trying to access array offset on null in netsession.net/httpdocs/modules/hardware/plugins/usrmgr_details_tab.php on line 11
PHP Warning: Trying to access array offset on null in netsession.net/httpdocs/modules/hardware/plugins/usrmgr_details_tab.php on line 12
PHP Warning: Trying to access array offset on null in netsession.net/httpdocs/modules/hardware/plugins/usrmgr_details_tab.php on line 13
PHP Warning: Trying to access array offset on null in netsession.net/httpdocs/modules/hardware/plugins/usrmgr_details_tab.php on line 14
PHP Warning: Trying to access array offset on null in netsession.net/httpdocs/modules/hardware/plugins/usrmgr_details_tab.php on line 15
PHP Warning: Trying to access array offset on null in netsession.net/httpdocs/modules/hardware/plugins/usrmgr_details_tab.php on line 16
PHP Warning: Trying to access array offset on null in netsession.net/httpdocs/modules/hardware/plugins/usrmgr_details_tab.php on line 17
PHP Warning: Trying to access array offset on null in netsession.net/httpdocs/modules/hardware/plugins/usrmgr_details_tab.php on line 18
PHP Warning: Trying to access array offset on null in netsession.net/httpdocs/modules/hardware/plugins/usrmgr_details_tab.php on line 19
PHP Warning: Trying to access array offset on null in netsession.net/httpdocs/modules/hardware/plugins/usrmgr_details_tab.php on line 20
PHP Warning: Trying to access array offset on null in netsession.net/httpdocs/modules/hardware/plugins/usrmgr_details_tab.php on line 21
PHP Warning: Trying to access array offset on null in netsession.net/httpdocs/modules/hardware/plugins/usrmgr_details_tab.php on line 22
PHP Warning: Trying to access array offset on null in netsession.net/httpdocs/modules/hardware/plugins/usrmgr_details_tab.php on line 30
PHP Warning: Trying to access array offset on null in netsession.net/httpdocs/modules/hardware/plugins/usrmgr_details_tab.php on line 33
PHP Warning: Undefined variable $plug_bttn_hw in netsession.net/httpdocs/modules/hardware/plugins/usrmgr_details_tab.php on line 33
```

**Comment by andygrunwald** [comment]
Fix waiting in https://github.com/lansuite/lansuite/pull/1101

**Comment by M4LuZ** [comment]
> Fix waiting in #1101

..for the lower part. 
For the part "pressing menu link start" some details are needed, as I cannot reproduce the issue on start / landing page
